docs/superpowers/plans/2026-05-22-nanosecond-timestamps.md

```
@@ -0,0 +1,2393 @@
+Expected: PASS — 7 tests pass.
+
+- [ ] **Step 7: Run lint**
+
+Run: `task lint`
+Expected: no violations in `internal/pgnanos/`.
+
+- [ ] **Step 8: Commit**
+
+Commit using VCS-appropriate commands per `references/vcs-preamble.md`. Suggested message:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)